libraries/I2S/src/I2S.h

```diff
@@ -110,14 +117,17 @@ class I2S : public Stream {
 private:
     pin_size_t _pinBCLK;
     pin_size_t _pinDOUT;
+    pin_size_t _pinMCLK; //RP
     int _bps;
     int _freq;
+    int _multMCLK;  // RP
     size_t _buffers;
     size_t _bufferWords;
     int32_t _silenceSample;
     bool _isLSBJ;
     bool _isOutput;
     bool _swapClocks;
+    bool _MCLKenabled = false; // RP
```

Comment:
Could we set this value in the constructor, not here?  We are setting the mclk multipler/pin/etc. there already.